### DISCUSSION TITLE
Facing problem when installing, always failed to download. Please help.

### DISCUSSION THREAD

**Discussion by Amarlam** [lllyasviel/Fooocus#2698] [Q&A] [answered]
![Screenshot (47)](https://github.com/lllyasviel/Fooocus/assets/114004239/dcd827dc-8e6e-4daa-b5a3-00e11f4ae1dd)

**Comment by Rogala** [comment, marked as answer]
Disable the proxy server if there is one, or even the firewall, try without them.

**Reply by mashb1t** [reply]
^ this, we've had this multiple times in discussions already.

**Reply by Amarlam** [reply]
Thank you bro for your help. There is proxy server on, after turning off, work fine.
